content/manager/bootstrapping.md

```diff
@@ -165,6 +165,153 @@ Services:
 {{< /gsHighlight >}}
 
 
+# Deploying a Manager Image
+
+Images are provided with all dependencies and the manager pre-installed for AWS and OpenStack. These allow you to get up and running with Cloudify with minimal user input required.
+
+(These images make sensible assumptions about how the manager is set up. If you want fine-grained control over your manager setup have a look at the [AWS](/manager/bootstrap-ref-aws) or [OpenStack](/manager/bootstrap-reference-openstack) bootstrapping guides instead).
+
+{{% gsNote title="Prerequisites" %}}
+ * Account credentials for the platform you are deploying on
+ * For the command-line install, the [`cfy` command](/intro/installation)
+{{% /gsNote %}}
+
+
+To run Cloudify Manager using an Image:
+
+ 1. Download an image from the [downloads page](http://getcloudify.org/downloads/get_cloudify.html)
+
+    {{% gsNote %}}
+    Choose the image that corresponds to your platform.
+    If you are uing AWS you can use the public AMI provided through the link above, skip to step 3.
```

Comment:
If you're using auto-numbering from markdown you probably want to provide a better anchor than the step number.